tex-nutshell/tex-nutshell.tex

```diff
@@ -458,8 +458,9 @@
 horizontal mode: the first character of a paragraph (most common
 situation) or \i indent `\indent`, \i noindent `\noindent`, \i hskip `\hskip` 
 (and its alternatives),
-\i vrule `\vrule`\fnote {The list is not fully completed, but most important commands
-are mentioned here.} and the plain \TeX/ macro \i leavevmode `\leavevmode`.
+\i vrule `\vrule` and the plain \TeX/ macro 
+\i leavevmode `\leavevmode`\fnote {The list is not fully completed, but most important commands
```

Comment:
While touching this, what do you think about:

`The list is not exhaustive, but most important commands are mentioned.`